docs/development/build.md

```diff
@@ -50,6 +44,6 @@ If you modify files in the /assets folder, you will need to rebuild the assets:
 
 ```
 $GOPATH/src/github.com/google/cadvisor $ ./build/assets.sh
-$GOPATH/src/github.com/google/cadvisor $ godep go build .
+$GOPATH/src/github.com/google/cadvisor $ make
```

Comment:
`make build` includes ./build/assets.sh, so the above line is redundant. I've been meaning to split out a separate `make assets` rule that build depends on...